python/vineyard/data/tests/test_arrow.py

```diff
@@ -66,21 +66,40 @@ def test_arrow_array(vineyard_client):
     assert vineyard_client.get(object_id).values.equals(nested_arr.values)
 
 
-def test_record_batch(vineyard_client):
+def test_arrow_array_with_rpc_client(vineyard_rpc_client):
+    test_arrow_array(vineyard_rpc_client)
+
+
+def test_record_batch_with_rpc_client(vineyard_rpc_client):
+    test_record_batch(vineyard_rpc_client)
+
+
+@pytest.mark.skip
+def build_record_batch():
     arrays = [
         pa.array([1, 2, 3, 4]),
         pa.array(['foo', 'bar', 'baz', None]),
         pa.array([True, None, False, True]),
     ]
     batch = pa.RecordBatch.from_arrays(arrays, ['f0', 'f1', 'f2'])
+    return batch
+
+
+def test_record_batch(vineyard_client):
+    batch = build_record_batch()
     _object_id = vineyard_client.put(batch)  # noqa: F841
     # processing tables that contains string is not roundtrip, as StringArray
     # will be transformed to LargeStringArray
     #
     # assert batch.equals(vineyard_client.get(object_id))
 
 
-def test_table(vineyard_client):
+def test_table_with_rpc_client(vineyard_rpc_client):
+    test_table(vineyard_rpc_client)
+
+
+@pytest.mark.skip
+def build_tabel():
```

Comment:
`tabel` -> `table`, and, with pytest, functions that don't starts with `test_` are not test cases and don't need the skip markers.